widgets/contact/contact.php

```diff
@@ -993,9 +998,9 @@ function contact_form_action( $instance, $storage_hash ) {
 				continue;
 			}
 			$field_name = $this->name_from_label( ! empty( $field['label'] ) ? $field['label'] : $i, $field_ids ) . '-' . $instance['_sow_form_id'];
-			$value      = ! empty( $post_vars[ $field_name ] ) ? $post_vars[ $field_name ] : '';
+			$value      = isset( $post_vars[ $field_name ] ) ? $post_vars[ $field_name ] : '';
 
-			if ( empty( $value ) ) {
+			if ( empty( $value ) && $value !== '0' ) {
```

Comment:
Is there not a function that covers all cases here? Like maybe `strlen`?

Reply:
Yeah, that'll work. I was being lazy. :finnadie: